docs/resources/security_identifier.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the security_identifier Resource
+platform: windows
+---
+
+# security_identifier
+
+Use the `security_identifier` InSpec resource to test the Security Identifier (SID) for users and groups on Windows..
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with InSpec itself. You can use it automatically.
+
+### Version
+
+This resource first became available in vX.Y.Z of InSpec.
+
+## Syntax
+
+A `security_identifier` resource should specify the name and type of the entity to test the SID for:
+
+    describe security_identifier(group: 'Everyone') do
+      its('sid') { should eq 'S-1-1-0' }
+    end
+
+where
+
+* `group:` specifies that `'Everyone'` should be a group. `user:` can be used to specify a user account.
+** It is necessary to declare the type of the entity because Windows allows users, groups and other entities to share names. If you really need to not specify the type, `unsupported:` can be used. This will attempt to match the name to a group and then a useraccount. This may take longer to execute and comes with the risk of Inspec matching the name to an unintended type of entity. 
```

Comment:
I think you mean 'unspecified', not 'unsupported'

Reply:
When referred to as a product, we try to refer to it as InSpec (note caps).  As a CLI or gem, use 'inspec'.